src/situations/accueil/vues/formulaire_identification.js

```diff
@@ -2,8 +2,9 @@ import 'accueil/styles/formulaire_identification.scss';
 import { traduction } from 'commun/infra/internationalisation';
 
 export default class FormulaireIdentification {
-  constructor (registreUtilisateur) {
+  constructor (registreUtilisateur, retourAccueil = () => { window.location.assign('/'); }) {
```

Comment:
plutôt que de faire une redirection, je propose d'utiliser `basculeAffichageFormulaireIdentification()` https://github.com/betagouv/competences-pro/blob/master/src/situations/accueil/vues/accueil.js#L63

On pourrait  déplacer la ligne 57 `$progression.css('background-image', `url('${this.depotRessources.progression(niveau).src}')`);` dans le bloc `else`

```
const basculeAffichageFormulaireIdentification = () => {
      if (!this.registreUtilisateur.estConnecte()) {
        formulaireIdentification.affiche($situations, $);
        boiteUtilisateur.supprime();
      } else {
        $progression.css('background-image', `url('${this.depotRessources.progression(niveau).src}')`);
        formulaireIdentification.supprime();
        boiteUtilisateur.affiche($titre, $);
      }
    };
```

Reply:
Oui, c'est une bonne idée… mais ça demande qu'on regarde aussi le raffraîchissement des accès aux bâtiments (ce dont je m'occupe).

Reply:
Done ✅ 😤 
`src/situations/accueil/vues/accueil.js` et `tests/situations/accueil/vues/accueil.js` sont suffisamment pénibles à utiliser pour mériter un coup de nettoyage. Je fais ça dans une PR à part.